config/environments/.env.template

```
@@ -69,6 +69,24 @@ RISK_DAILY_LOSS_LIMIT_PCT=0.05
 RISK_KILL_SWITCH_ENABLED=0
 RISK_REDUCE_ONLY_MODE=0
 
+# -----------------------------------------------------------------------------
+# RISK (RELIABILITY / GRACEFUL DEGRADATION)
+# -----------------------------------------------------------------------------
+# Read via the RISK_ prefix in RiskConfig.from_env(); see docs/RELIABILITY.md and
+# src/gpt_trader/features/live_trade/risk/config.py. Values below are the defaults.
+RISK_API_ERROR_RATE_THRESHOLD=0.2
+RISK_API_RATE_LIMIT_USAGE_THRESHOLD=0.9
+RISK_API_HEALTH_COOLDOWN_SECONDS=300
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RISK_API_HEALTH_COOLDOWN_SECONDS key should go before the RISK_API_RATE_LIMIT_USAGE_THRESHOLD key

(UnorderedKey)